assign-doi-datacite/src/test/java/no/unit/nva/datacite/models/DataCiteMdsClientSecretConfigTest.java

```diff
@@ -0,0 +1,45 @@
+package no.unit.nva.datacite.models;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.Test;
+
+public class DataCiteMdsClientSecretConfigTest {
+
+
+    public static final String INSTITUTION = "institution";
+    public static final String INSTITUTION_PREFIX = "institutionPrefix";
+    public static final String DATA_CITE_MDS_CLIENT_URL = "dataCiteMdsClientUrl";
+    public static final String DATACITE_MDS_CLIENT_USERNAME = "dataCiteMdsClientUsername";
+    public static final String DATACITE_MDS_CLIENT_PASSWORD = "dataCiteMdsClientPassword";
+
+    @Test
+    public void testConstructor() {
+        var secretConfig = new DataCiteMdsClientSecretConfig(INSTITUTION,
```

Comment:
`assertThat(secretConfig, doesNotHaveNullOrEmptyValues())`

Reply:
I keep forgetting about this one! Thanks!